.github/workflows/whalesay.yaml

```diff
@@ -0,0 +1,58 @@
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+  push:
+    branches:
+      - main
```

Comment:
Filters go up here:

```
on:
  pull_request:
    types:
      - opened
      - synchronize
    paths:
      - './.github/workflows/whalesay.yaml'
      - './.github/workflows/pr-and-release-container.yaml'
      - './.github/actions/pr-container.yaml'
      - './.github/actions/release-container.yaml'
      - './containers/whalesay/**'
  push:
    branches:
      - main
    paths:
      - './.github/workflows/whalesay.yaml'
      - './.github/workflows/pr-and-release-container.yaml'
      - './.github/actions/pr-container.yaml'
      - './.github/actions/release-container.yaml'
      - './containers/whalesay/**'
```

You can see it's just global filters for the whole workflow run, we don't get to sub-key them for individual containers or charts like we do with the action we were calling.

When we were building multiple assets in the same workflow run we needed that ability to filter out different sets of changes. But if we're doing one workflow per asset that asset can use these global filters as its own.